local_deploy/encode_query_aio.py

```diff
@@ -0,0 +1,128 @@
+import ast
```

Comment:
is it possible to use `square-model-client` package instead of these functions?

Reply:
Done in [e3ed653](https://github.com/UKP-SQuARE/square-core/pull/354/commits/e3ed6536331a5d2582b2207a0efa929f79b8bee1)